drivers/tee/optee/optee.c

```diff
@@ -449,6 +511,31 @@ static void free_shm_pages(void **pages)
 	}
 }
 
+static void handle_rpc_supp_cmd(const struct device *dev, struct optee_msg_arg *arg)
+{
+	// TODO: FIX HARDCODE
+	struct tee_param params[5];
```

Comment:
Hm. It's not clear to me why have you made this hardcode? IIUC we have num_params passed in arg argument.
At least if there is a good reason why have you hardcoded this param I'd like to see it in TODO.